package.json

```diff
@@ -41,14 +41,18 @@
     "eslint-plugin-import": "^2.14.0",
     "jwt-decode": "^2.2.0",
     "materialize-css": "^1.0.0-rc.2",
+    "medium-editor": "^5.23.3",
     "npm-watch": "^0.4.0",
     "prop-types": "^15.6.2",
     "query-string": "^6.2.0",
     "react": "^16.6.0",
     "react-dom": "^16.6.0",
+    "react-idle-timer": "^4.0.9",
     "react-materialize": "^2.4.8",
```

Comment:
medium-editor for text-area tag.
react-idle-timer for checking the idle time of the user before calling the auto-save function
react-tag-input for adding tags